internal/patch.js

```diff
@@ -7,19 +7,23 @@ module.exports = () => {
   Array.prototype[fl.equals] = function(y) {
     return this.length === y.length && this.join('') === y.join('');
   };
-  Array.prototype[fl.map] = Array.prototype.map;
+  Array.prototype[fl.map] = function(f) {
+    return this.map(x => f(x));
+  };
```

Comment:
Will do. :)

Sanctuary is missing some of the laws, as we've only defined tests for those applicable to Maybe and/or Either. sanctuary-js/sanctuary#323 adds tests for Alt, Plus, and Alternative. Once that pull request has been merged I'll create __fantasyland/fantasy-laws__ and submit an initial pull request.